packages/theme/src/core/pagination/types.ts

```diff
@@ -1,5 +1,8 @@
+import { FontVariants } from "../font/types";
+
 export interface PaginationTheme {
     /** Theme for page number button */
+    fontVariant?: FontVariants;
```

Comment:
Can you please move this line above the comment and also could you please add comment for `fontVariant`  also. Because these comments are being showed in Theme table in storybook.

Reply:
Could you please add default values of these types in the index file also.

Reply:
updated accordingly